src/mgr/Action.cc

```diff
@@ -120,3 +119,20 @@ Mgr::Action::fillEntry(StoreEntry* entry, bool writeHttpHeader)
         entry->complete();
 }
 
+void
+Mgr::openKidSection(StoreEntry *entry, const bool is_yaml)
+{
+    if (is_yaml)
+        storeAppendPrintf(entry, "---\nkid: %d\n", KidIdentifier);
+    else
+        storeAppendPrintf(entry, "by kid %d {\n", KidIdentifier);
+}
+
+void
+Mgr::closeKidSection(StoreEntry *entry, const bool is_yaml)
+{
+    if (is_yaml)
+        storeAppendPrintf(entry, "...\n");
+    else
+        storeAppendPrintf(entry, "} by kid %d\n", KidIdentifier);
```

Comment:
See the other change request. Ignore this one if there is something to discuss there.
```suggestion
        storeAppendPrintf(entry, "} by kid%d\n\n", KidIdentifier);
```